openmdao/docs/openmdao_book/features/model_visualization/real_time_optimization_progress_plot.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ee8e8873",
+   "metadata": {},
+   "source": [
+    "# Using the Real-time Optimization Progress Plot\n",
+    "\n",
+    "The real-time optimization progress plot feature allows the user to see, in real-time, the progress of an optimization run by plotting design variables, constraints, and the objective.\n",
+    "\n",
+    "There two different ways that the plot can be generated."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c48021c2",
+   "metadata": {},
+   "source": [
+    "## Running the plot as part of an optimization run\n",
+    "\n",
+    "Likely the most common way users will use this feature is to run the optimization script is using the `opendmao rtplot` subcommand.\n",
+    "\n",
+    "```\n",
+    "openmdao rtplot path_to_script.py\n",
+    "```\n",
+    "\n",
+    "This will run the optimization script and also automatically launch the plotting app in a browser window at the appropriate time during the optimization run. \n",
+    "\n",
+    "The plot will then be periodically be updated as the results of driver iterations complete. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5acdecaf",
+   "metadata": {},
+   "source": [
+    "```{important}\n",
+    "Plotting of the driver iterations will only work if the script has a case recorder attached to the `Driver`.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "10ba0143",
+   "metadata": {},
+   "source": [
+    "## Running the plot separate from an optimization run\n",
+    "\n",
+    "This is used when you want to directly point the plotting application at a case recorder file that contains driver iteration information. \n",
+    "\n",
+    "For example, let's say you have an optimization script named `optimize_system.py`. In that script, a case recorder is attached to the `Driver` like this:\n",
+    "\n",
+    "```\n",
+    "prob.driver.add_recorder(SqliteRecorder(\"system_driver_recording.sql\"))\n",
+    "```\n",
+    "if you run that script:\n",
+    "\n",
+    "```\n",
+    "python optimize_system.py\n",
+    "```\n",
+    "\n",
+    "The result will be a case recorder file in \n",
+    "\n",
+    "```\n",
+    "optimize_system_out/system_driver_recording.sql\n",
+    "\n",
+    "```\n",
+    "\n",
+    "Then to see a plot of the optimization progress, or see the results after the script is complete:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ac29bfc",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1860288",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "openmdao realtime_opt_plot optimize_system_out/system_driver_recording.sql\n",
```

Comment:
_realtime_ is really a misnomer for this version of the command.. shouldn't it just be `opt_plot`?  ;)